indexing-hadoop/src/main/java/org/apache/druid/indexer/JobHelper.java

```diff
@@ -418,8 +402,19 @@ public static boolean runJobs(List<Jobby> jobs, HadoopDruidIndexerConfig config)
       }
     }
 
+    return succeeded;
+  }
+
+  public static void maybeDeleteIntermediatePath(
+      boolean indexerGeneratorJobSucceeded,
```

Comment:
I think this should just be `jobSucceeded` since it's used by more than one job

Reply:
fixed